pkg/transforms/argoapplication.go

```diff
@@ -73,16 +98,61 @@ func ArgoApplicationResourceBuilder(a *ArgoApplication) *ArgoApplicationResource
 	node.Properties["chart"] = a.Spec.Source.Chart
 	node.Properties["repoURL"] = a.Spec.Source.RepoURL
 	node.Properties["targetRevision"] = a.Spec.Source.TargetRevision
+
 	if a.Spec.Source.TargetRevision == "" {
 		node.Properties["targetRevision"] = "HEAD"
 	}
 
 	// Status properties
-	node.Properties["status"] = a.Status.Health.Status
+	node.Properties["healthStatus"] = a.Status.Health.Status
+	node.Properties["syncStatus"] = a.Status.Sync.Status
+
+	// resource list properties
+	resourcesByte, err := yaml.Marshal(a.Status.Resources)
+	if err != nil {
+		glog.Errorf("Marshaling ArgoCD application %v/%v resource list failed: %v", a.Namespace, a.Name, err)
+	} else {
+		node.Properties["resources"] = string(resourcesByte)
```

Comment:
Same as comment https://github.com/stolostron/search-collector/pull/226/files#r1013266422

The `resources` property is encoding too much information, which will make the search functionality difficult.

In this case, I'd prefer to implement this as an edge to the resource. This will fit better with the relationship model used by search and avoid duplicate information.